pages/_template.jsx

```diff
@@ -2,43 +2,77 @@ import React, { Component, PropTypes } from 'react';
 import Headroom from 'react-headroom';
 import { Link } from 'react-router';
 import { prefixLink } from 'gatsby-helpers';
+import classNames from 'classnames';
 
 require('css/_index.scss');
+
+const navbar_height = 54;
```

Comment:
Variables in Javascript should be camelCase instead of snake_case. There's nothing wrong with snake_case, it's just most style guides use camelCase.

I mix and match [Airbnb's Style Guide](https://github.com/airbnb/javascript) and [Khan Academy's](https://github.com/Khan/style-guides)